frontend/src/app/services/user.service.ts

```diff
@@ -50,6 +51,15 @@ export class UserService {
   }
 
   setIsOkrChampion(user: User, isOkrChampion: boolean) {
-    return this.httpClient.put(`${this.API_URL}/${user.id}/isokrchampion/${isOkrChampion}`, {});
+    return this.httpClient.put(`${this.API_URL}/${user.id}/isokrchampion/${isOkrChampion}`, {}).pipe(
+      tap(() => {
+        this.reloadUsers();
+        this.reloadCurrentUser().subscribe();
+      }),
+    );
+  }
+
+  createUsers(userList: NewUser[]) {
+    return this.httpClient.post<User>(`${this.API_URL}/createall`, userList).pipe(tap(() => this.reloadUsers()));
```

Comment:
Wir machen nichts mit den vom endpoint zurückgelieferten Benutzer. Ich denke entweder sollten wir den Returnwert des Endpoints entfernen oder das `.pipe(tap(() => this.reloadUsers()));` anpassen, damit die neuen benutzer in die userliste gepushed werden und das BehaviourSubject getriggert wird.

Reply:
Ich glaube Variante 1 ist besser, weil die Entität NewUser ein paar Attribute weniger hat als die, die wir in der Liste benötigen.